markdownhighlighter.cpp

```diff
@@ -811,86 +819,121 @@ void MarkdownHighlighter::highlightSyntax(const QString &text)
-        case HighlighterState::CodeBash :
-            loadShellData(types, keywords, builtin, literals, others);
-            comment = QLatin1Char('#');
-            break;
-        case HighlighterState::CodePHP :
-        case HighlighterState::CodePHPComment :
-            loadPHPData(types, keywords, builtin, literals, others);
-            break;
-        case HighlighterState::CodeQML :
-        case HighlighterState::CodeQMLComment :
-            loadQMLData(types, keywords, builtin, literals, others);
-            break;
-        case HighlighterState::CodePython :
-            loadPythonData(types, keywords, builtin, literals, others);
-            comment = QLatin1Char('#');
-            break;
-        case HighlighterState::CodeRust :
-        case HighlighterState::CodeRustComment :
-            loadRustData(types, keywords, builtin, literals, others);
-            break;
-        case HighlighterState::CodeJava :
-        case HighlighterState::CodeJavaComment :
-            loadJavaData(types, keywords, builtin, literals, others);
-            break;
-        case HighlighterState::CodeCSharp :
-        case HighlighterState::CodeCSharpComment :
-            loadCSharpData(types, keywords, builtin, literals, others);
-            break;
-        case HighlighterState::CodeGo :
-        case HighlighterState::CodeGoComment :
-            loadGoData(types, keywords, builtin, literals, others);
-            break;
-        case HighlighterState::CodeV :
-        case HighlighterState::CodeVComment :
-            loadVData(types, keywords, builtin, literals, others);
-            break;
-        case HighlighterState::CodeSQL :
-            loadSQLData(types, keywords, builtin, literals, others);
-            break;
-        case HighlighterState::CodeJSON :
-            loadJSONData(types, keywords, builtin, literals, others);
-            break;
-        case HighlighterState::CodeXML :
-            xmlHighlighter(text);
-            return;
-        case HighlighterState::CodeCSS :
-        case HighlighterState::CodeCSSComment :
-            isCSS = true;
-            loadCSSData(types, keywords, builtin, literals, others);
-            break;
-        case HighlighterState::CodeTypeScript:
-        case HighlighterState::CodeTypeScriptComment:
-            loadTypescriptData(types, keywords, builtin, literals, others);
-            break;
-        case HighlighterState::CodeYAML:
-            isYAML = true;
-            comment = QLatin1Char('#');
-            loadYAMLData(types, keywords, builtin, literals, others);
-            break;
-        case HighlighterState::CodeINI:
-            iniHighlighter(text);
-            return;
-        case HighlighterState::CodeTaggerScript:
-             taggerScriptHighlighter(text);
-             return;
-        case HighlighterState::CodeVex:
-            loadVEXData(types, keywords, builtin, literals, others);
-            break;
+    case HighlighterState::CodeCpp:
```

Comment:
Is there a reason you changed the switch indent?

Reply:
That was Qt Creator. It just changes the indent everytime i make a change, and this one involved a lot of changes so everything in the switch is one \t back

Reply:
It looks uuuuugly and so far I used the previous styling everywhere.

Reply:
Yes I know. Reverted.
I just don't know why the editors always revert it. :laughing:

Reply:
Thank you :grin: